src/main/java/seedu/address/logic/parser/GenerateResumeCommandParser.java

```diff
@@ -0,0 +1,35 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.generate.GenerateResumeCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.item.field.Name;
+
+/**
+ * Parses input arguments and creates a new GenerateResumeCommand object
+ */
+public class GenerateResumeCommandParser implements Parser<GenerateResumeCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the GenerateResumeCommand
+     * and returns a GenerateResumeCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public GenerateResumeCommand parse(String args) throws ParseException {
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_NAME);
+        if (argMultimap.getPreamble().isEmpty()) {
+            throw new ParseException(GenerateResumeCommand.MESSAGE_USAGE);
```

Comment:
I think the new convention is that we'll wrap this around a "Invalid Command Format!\n...." can look at my PR to see? Not all but whenever we call the XYZ.Command.MESSAGE_USAGE it's quite clear